src/main/java/com/yugabyte/sample/apps/AppBase.java

```diff
@@ -179,6 +179,16 @@ protected Connection getPostgresConnection(String database, String username, Str
         ContactPoint contactPoint = getRandomContactPoint();
         Properties props = new Properties();
         props.setProperty("user", username);
+        // Check if the smart driver is there in the class path
+        boolean smartDriver = true;
+        try {
+          Class.forName("com.yugabyte.ysql.ClusterAwareLoadBalancer");
+        } catch (ClassNotFoundException cnfe) {
```

Comment:
There are more than one exception which may be thrown from forName().

https://docs.oracle.com/javase/7/docs/api/java/lang/Class.html#forName(java.lang.String)

Should the other exceptions be covered ?

Reply:
The other two exceptions are not necessary to be handled.
'LinkageError': As there is no dependency on any other class for ClusterAwareLoadBalancer to be initialised.
'ExceptionInInitializerError' - No static initialiser block in the class.